examples/rbac/server/feature_repo/permissions_apply.py

```diff
@@ -0,0 +1,33 @@
+from feast import FeatureStore
+from feast.feast_object import ALL_RESOURCE_TYPES
+from feast.permissions.action import QUERY, AuthzedAction, ALL_ACTIONS
+from feast.permissions.decision import DecisionStrategy
+from feast.permissions.permission import Permission
+from feast.permissions.policy import RoleBasedPolicy
+
+store = FeatureStore(repo_path="")
+
+admin_roles = ["feast-admin-role"]
+user_roles = ["feast-user-role"]
+
+user_perm = Permission(
+    name="feast_user_permission",
+    types=ALL_RESOURCE_TYPES,
+    with_subclasses=False,
+    policy=RoleBasedPolicy(roles=user_roles),
+    actions=[AuthzedAction.READ] + QUERY
+)
+
+admin_perm = Permission(
+    name="feast_admin_permission",
+    types=ALL_RESOURCE_TYPES,
+    with_subclasses=False,
+    policy=RoleBasedPolicy(roles=admin_roles),
+    actions=ALL_ACTIONS
+)
+
+Permission.set_global_decision_strategy(DecisionStrategy.AFFIRMATIVE)
```

Comment:
be aware that this option does not work until next Theo's PR.
my suggestion is to add both list of roles to `user_perm` for now